tests/acceptance/settings/settings.json

```diff
@@ -0,0 +1,32 @@
+{
+    "environment": {
+        "name": "experimentation"
+    },
+    "facts_service": {
+        "protocol": "http",
+        "host": "1.2.3.4",
+        "port": "5000",
+        "resource": "/v1.0",
+        "os_tenant_id": "00000000000000000000000000000000",
+        "host_user": "root",
+        "host_password": "",
+        "private_key_location": "",
+        "service_log_path": "/var/logs/fiware_facts",
```

Comment:
I do not know if this path should be the real. In that case, path is /var/log/policyManager/fiware-facts and /var/log/policyManager/fiware-cloto